### ISSUE TITLE
Support Math equation label and numbering.

### ISSUE THREAD

**Issue by gdolle** [GitbookIO/gitbook#1203]
It would be very useful to support equation reference using label and display numbering.

```
$$
\lambda+\mu
\label{mylabel}
$$

I cite my equation \eqref{mylabel} in the text.
```

Mathjax support that syntax http://mathjax.readthedocs.org/en/latest/tex.html#automatic-equation-numbering.

**Comment by user-don** [comment]
Would love to see this as well!
